evm/src/cpu/kernel/mod.rs

```diff
@@ -1,14 +1,14 @@
 pub mod aggregator;
 pub mod assembler;
 mod ast;
+pub mod interpreter;
```

Comment:
Hm is the idea to use `Interpreter` to help with witness generation? (If so I'm not sure it's useful there, but we can discuss later.)

Reply:
Oh I think this was just to silence a bunch of dead code warnings. I changed it back to a `cfg(test)` module.